packages/x-components/src/x-modules/search/store/types.ts

```diff
@@ -32,7 +32,7 @@ export interface SearchState extends StatusState {
   /** A flag to indicate if new results are append to the current instead of replacing them. */
   isAppendResults: boolean;
   /** The origin property of the request. */
-  origin: QueryOrigin | null;
+  origin: Origin | null;
```

Comment:
Why here is `null` but later(line 109) is `undefined`?

Reply:
Good catch, it should be `null`